doc/source/multivariate.rst

```diff
@@ -144,12 +144,12 @@ We realize that the mean vector and the covariance often have special forms in p
 
 .. code-block:: julia
 
-    immutable MvNormal{Cov<:AbstractPDMat,Mean<:Union{Vector{Float64},ZeroVector{Float64}}} <: AbstractMvNormal
+    immutable MvNormal{Cov<:AbstractPDMat,Mean<:Union{Vector,ZeroVector}} <: AbstractMvNormal
```

Comment:
Just to make sure I understand, something like
```julia
immutable MvNormal{Cov<:AbstractPDMat, Mean<:Union{Vector,ZeroVector, T<:Real} <: AbstractMvNormal
```
and then outer constructors set `T = promote_type(eltype(μ), eltype(Σ))`?

Right now, μ and Σ can have different element types.